bin/console

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env ruby
```

Comment:
Is the purpose of this class to have a rails-like console session? If so should we add it to the readme? :grin: